app/controllers/taggings_controller.rb

```diff
@@ -0,0 +1,18 @@
+class TaggingsController < ApplicationController
+  def new
```

Comment:
if we're doing the actual creating of the object in the database here, then this should be `create` instead of `new` 👍